packages/network/src/base/msg.rs

```diff
@@ -536,7 +542,7 @@ mod tests {
         assert_eq!(header.feature, 2);
         assert_eq!(header.meta, 3);
         assert_eq!(header.route, RouteRule::Direct);
-        assert_eq!(header.encrypt, true);
+        assert!(header.encrypt);
```

Comment:
I prefer assert_eq because it has the same prefix as the other lines. It will look prettier.

Reply:
I can understand that 😁 

Will revert that.